tests/installation/test_installer.py

```diff
@@ -1283,6 +1283,49 @@ def test_run_installs_with_local_poetry_directory_transitive(
     assert installer.executor.installations_count == 6
 
 
+def test_run_installs_with_local_poetry_directory_transitive_no_path(
```

Comment:
Can you integrate this with `pytest.mark.parametrize` in the previous test?

`skip_directory=False` -> `directory_installs`, `installations_count == 6`
`skip_directory=True` -> `not directory_installs`, `installations_count == ...`

Reply:
did it